tests/network.py

```diff
@@ -10,6 +11,23 @@
 """
 
 BRIDGE_NF_CALL_IPTABLES = "/proc/sys/net/bridge/bridge-nf-call-iptables"
+COMMAND_TIMEOUT = 60
+
+
+@contextmanager
+def managed_nspopen(*args, **kwds):
+    proc = NSPopen(*args, **kwds)
+    try:
+        yield proc
+    finally:
+        if proc.poll() is None:
+            # send SIGKILL to the process and wait for it to die if it's still
+            # running
+            proc.kill()
+            proc.communicate()
+        # release proxy process resourecs
+        proc.release()
```

Comment:
Given https://docs.pyroute2.org/netns.html?highlight=nspopen#pyroute2.netns.process.proxy.NSPopen I think we should do 

``` Python
    proc.wait()
    proc.release()
```

What do you think?

Reply:
Doesn't seem very useful given that the process is known not be running at this point: https://docs.python.org/3/library/subprocess.html#subprocess.Popen.wait

Reply:
I see it now, thanks...